mkt/extensions/views.py

```diff
@@ -197,6 +205,12 @@ def check_permissions(self, request):
         super(ExtensionVersionViewSet, self).check_permissions(request)
 
         extension = self.get_extension_object()
+        # You can't modify or create versions on disabled extensions, even if
+        # you are the owner.
+        if extension.disabled and self.request.method not in SAFE_METHODS:
+            raise PermissionDenied(
+                _(u'Modifying or submitting versions is forbidden for disabled'
+                  u' Add-ons.'))
```

Comment:
We typically don't. But this message could end up being shown by the frontend, since it can happen at version creation time, so I thought it'd be best to translate it.